src/Service/HashingService.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+namespace QuizApp\Service;
+
+
+class HashingService
+{
+    public function hash(string $password): ?string
+    {
+        return password_hash($password, PASSWORD_BCRYPT);
```

Comment:
pass that constant as an argument to the constructor of the service, in order to be able to easily change it. This is the lightest version of a Strategy pattern (though not really) :D